system/lib/libcxx/include/__config

```diff
@@ -10,6 +10,18 @@
 #ifndef _LIBCPP_CONFIG
 #define _LIBCPP_CONFIG
 
+// XXX Emscripten: begin site config
+#ifdef __EMSCRIPTEN__
+#define _LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS
+// Normally macros like _LIBCPP_HAS_NO_THREADS are fixed at build time,
+// but we to have this header work for both the multi-threaded and
+// single-threaded version of libc++.
+#ifndef _REENTRANT
```

Comment:
what is `_REENTRANT`?

can we not use the normal way to detect atomics in wasm?

Reply:
What is the normal way?    The _REENTRANT  macro is defined by clang when building with `-pthread`.

Reply:
We use `__EMSCRIPTEN_PTHREADS__` in other places to check for this.

I wasn't aware of `_REENTRANT`. Not opposed to it, but if we can be consistent that sounds better?

Reply:
Done